spec/src/main/asciidoc/core/EnterpriseBeanEnvironment.adoc

```diff
@@ -3200,4 +3200,4 @@ concurrent access to the SessionContext object to be thread-safe.
 The container
 must follow the requirements for all other resources and configuration
 parameters specified in the Jakarta EE Platform specification <<a9861>>.
-----
+----
```

Comment:
I meant the line with ----, if you build the docs everything from Chapter 11 is broken.

Reply:
Yes I see it now. I've fixed it and validated that the spec document is now generated with Chapter 11 and further included